### PR TITLE
remove anonymous function to ensure backwards compatibility

### PR DESCRIPTION
**Pull request by andersone1**
Remove anonymous function to ensure backwards compatibility

### CODE REVIEW

File: R/v.R [hunk 1]
```diff
@@ -129,7 +129,7 @@ v <- function(
 
   # Round numeric columns to 3 decimal places
   if(!is.null(.digits)){
-    .df <- .df %>% dplyr::mutate(across(where(is.numeric),\(x) prettyNum2(x, .digits)))
+    .df <- .df %>% dplyr::mutate(across(where(is.numeric), ~prettyNum2(.x, .digits)))
```
Comment (barrettk) [inline R/v.R:132]: I initially had it this way, but changed it to an anonymized function after getting some warning messages during `devtools::test()`. If you run `pkgr install`, do you get these warnings?

**Review by barrettk — COMMENTED** [review]
Left a comment before approving

File: R/v.R (hunk 1 above)
Comment (barrettk) [reply R/v.R:132]: If not happy to approve

**Review by barrettk — APPROVED** [review]
Looks good. I ran `devtools::test()` on my end and didn't get any warnings. Perhaps I had it a little different when I was getting them before

**Comment by andersone1** [comment]
> Looks good. I ran `devtools::test()` on my end and didn't get any warnings. Perhaps I had it a little different when I was getting them before

@barrettk OK cool, thank you for being thorough there. I will merge.

**Pull request merged**